FirstProject/css/main.css

```diff
@@ -80,11 +84,66 @@ h2.description_section {
     margin-left: 300px;
 }
 
-/*.row_section {*/
-    /*margin-left:254px;*/
-    /*margin-right: 255px;*/
-/*}*/
-
 fieldset.field_section {
     text-align: center;
-}
\ No newline at end of file
+}
+
+legend {
+    text-align: left;
+    color: #03A9F4;
+}
+
+input[type='radio']{
```

Comment:
I usually use double quotes too, but in this case I took the code from Internet and I let it like that.
Thanks for the suggestion.